back/scripts/enrichment/marches_enricher.py

```diff
@@ -20,5 +20,77 @@ def get_input_paths(cls, main_config: dict) -> typing.List[Path]:
     @classmethod
     def _clean_and_enrich(cls, inputs: typing.List[pl.DataFrame]) -> pl.DataFrame:
         marches, *_ = inputs
-        # Data analysts, please add your code here!
+        # TODO casting temporaire de titulaire_id en String, il faudra modifier le fichier source
+        marches = marches.with_columns(
+            pl.col("titulaire_id").cast(pl.String).alias("titulaire_id")
```

Comment:
pas besoin de l'alias car le non de la colonne reste le même :) idem je pense que l'on peut se passer de tous les  `.alias("titulaire_typeIdentifiant")` plus bas
```suggestion
            pl.col("titulaire_id").cast(pl.String)
```

Reply:
C'est bon